tidb-cloud/monitor-tidb-cluster.md

```diff
@@ -59,11 +59,26 @@ In TiDB Cloud, you can view the commonly used metrics of a cluster from the foll
 
 ### Metrics on the cluster overview page
 
-The cluster overview page provides general metrics of a cluster, including total QPS, query duration, active connections, TiDB CPU, TiKV CPU, TiFlash CPU, TiDB memory, TiKV memory, TiFlash memory, TiKV used storage size, and TiFlash used storage size.
-
-> **Note:**
->
-> Some of these metrics might be available only for Dedicated Tier clusters.
+The cluster overview page provides general metrics of a cluster.
+
+For dedicated Tier:
+| Metric name  | Labels | Description                                   |
+| :------------| :------| :-------------------------------------------- |
+| QPS | total | The number of SQL statements executed per second in all TiDB instances. |
+| Query Duration | avg | The duration from receiving a request from the client to TiDB till TiDB executing the request and returning the result to the client. |
+| Active Connection | active connections | The number of active connections to all TiDB instances. |
+| CPU | avg-tidb, avg-tikv, avg-tiflash | The average statistics of CPU usage for all TiDB instances. |
+| Memory | avg-tidb, avg-tikv, avg-tiflash | TThe average statistics of Memory usage for all TiDB instances. |
+| Used Storage Size | total-tikv, total-tiflash | The total storage size on TiKV or TiFlash instances. |
+
+
+For Serverless Tier:
+| Metric name  | Labels | Description                                   |
+| :------------| :------| :-------------------------------------------- |
+| QPS | total | The number of SQL statements executed per second in Serverless Tier. |
+| Query Duration | avg | The duration from receiving a request from the client to TiDB till TiDB executing the request and returning the result to the client. |
+| Active Connection | active connections | The number of active connections to Serverless Tier. |
+| Used Storage Size | total-tikv, total-tiflash | The total storage size on TiKV or TiFlash instances. |
```

Comment:
We don't have total-tiflash at this time, but it will be available soon. Are we going to put it here right now, or create another PR when it's ready?

Another thing we need to be careful, there is another storage size gauge in clusters list page. We probably will have to figure out a way to add TiFlash there.

Reply:
OK. we can add it after released.

Reply:
I delete the description about tiflash